Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR_test.py

```diff
@@ -644,6 +644,38 @@ def test_fetch_incidents__one_alert_in_response(mocker):
 ]
 
 
+def test_fetch_incidents_duplicates_len_is_limit(mocker):
```

Comment:
I want to improve the test to mock the bug. Please simulate the returning incidents to include the last run incidents, and the limit as well. Please verify the new incidents have been fetched and print an example before you change, so that the test fails. e.g. in the last run we have id=1,2 and the limit is 2, the search return id=1,2,3 so I want it to retrieve num 3 and fail to do so before the changes.